apps/mobile/src/features/account-selector-sheet.tsx

```diff
@@ -50,38 +50,36 @@ export function AccountSelectorSheet({ modalRef }: { modalRef: RefObject<BottomS
   const onAccountPress = useCallback(
     (accountId: string) => {
       const { fingerprint, accountIndex } = destructAccountIdentifer(accountId);
-      modalRef.current?.close();
+      sheetRef.current?.close();
       router.navigate({
         pathname: APP_ROUTES.WalletWalletsSettingsConfigureAccount,
         params: { wallet: fingerprint, account: accountIndex },
       });
     },
-    [router, modalRef]
+    [router, sheetRef]
   );
 
   return (
-    <Modal shouldHaveContainer={false} ref={modalRef}>
-      <BottomSheetScrollView ref={scrollViewRef}>
-        <Box p="5" gap="5">
-          {accounts.map((account, idx) => {
-            return (
-              <Draggable
-                idx={idx}
-                direction={direction}
-                scrollViewRef={scrollViewRef}
-                placeholderIdx={placeholderIdx}
-                cardsLength={accounts.length}
-                key={account.id}
-                cardId={account.id}
-                onCardPress={onAccountPress}
-                swapCardIndexes={swapAccountIndexes}
-              >
-                <AccountCard Icon={getAvatarIcon(account.icon)} name={account.name} />
-              </Draggable>
-            );
-          })}
-        </Box>
-      </BottomSheetScrollView>
-    </Modal>
+    <Sheet isScrollView shouldHaveContainer={false} ref={sheetRef} themeVariant={themeVariant}>
```

Comment:
@edgarkhanzadian I changed this to pass `isScrollView`, can you help double check its not broken? Thanks!